core-scala/src/test/scala/com/baeldung/scala/enumerations/FingersOperationTest.scala

```diff
@@ -0,0 +1,41 @@
+package com.baeldung.scala.enumerations
+
+import Fingers._
+import org.junit.Assert.assertTrue
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class FingersOperationTest {
```

Comment:
For consistency with other test classes/files, and as per naming convention, let's rename this to FingersOperationUnitTest.

Reply:
Thanks, done!